src/main/java/duke/controller/Main.java

```diff
@@ -0,0 +1,34 @@
+package duke.controller;
+
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+
+/**
+ * Encapsulates the GUI for the program.
+ */
+public class Main extends Application {
+    /**
+     * Starts the GUI welcome window.
+     *
+     * @param window the window of the current view.
+     */
+    @Override
+    public void start(Stage window) {
+        // execute GUI loop
+        try {
+            FXMLLoader fxmlLoader = new FXMLLoader(Main.class.getResource("/view/Menu.fxml"));
+            AnchorPane ap = fxmlLoader.load();
```

Comment:
Should this be named with anchorPane instead of ap to be consistent with the naming above and below this line.

Reply:
yes haha. I just directly copied from the tutorial. Nice spot.